source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -126,7 +126,7 @@
         var self = this;
         this.userView = data.UserView;
         this.userLogged = data.UserLogged;
-        this.hasAverageColumn = !data.Evaluation.Finished && (this.userView == 1 || this.userView == 3);
```

Comment:
Also, you're showing an average for evaluators too? What will that average consist of? (Since evaluators will see it before other evaluators submit their options or before the responsible does.)